pages/_wiki/newhere.md

```diff
@@ -0,0 +1,24 @@
+---
+permalink: /wiki/newhere/
```

Comment:
`permalink` is no longer necessary for new pages. It'll be deduced from the file name and path.

Reply:
removed that.